meta-ostro/recipes-swupd/clr-systemd-config/files/0002-systemd-service-for-updating-the-efi-combo.patch

```diff
@@ -1,40 +0,0 @@
-From b6828d8b0214e781c420357c4955e0c14ab0993b Mon Sep 17 00:00:00 2001
-From: Igor Stoppa <igor.stoppa@intel.com>
-Date: Thu, 28 Jan 2016 22:46:03 +0200
-Subject: [PATCH] systemd service for updating the efi combo
-
-The service propagates to the EFI partition the changes to
-the efi-combo file present in the rootfs.
-
-Signed-off-by: Igor Stoppa <igor.stoppa@intel.com>
----
- system/efi-combo-trigger.service | 7 +++++++
- system/update-triggers.target    | 1 +
- 2 files changed, 8 insertions(+)
- create mode 100644 system/efi-combo-trigger.service
-
-diff --git a/system/efi-combo-trigger.service b/system/efi-combo-trigger.service
-new file mode 100644
-index 0000000..a45f342
---- /dev/null
-+++ b/system/efi-combo-trigger.service
-@@ -0,0 +1,7 @@
-+[Unit]
-+Description=Updates the EFI combo application
-+BindsTo=update-triggers.target
-+
-+[Service]
-+Type=oneshot
-+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
should this be in some hidden namespace @igor-stoppa? we do not need this to end in $PATH, do we?